src/mesh/mesh.cpp

```diff
@@ -85,7 +86,10 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Packages_t &packages,
       lb_manual_(), nslist(Globals::nranks), nblist(Globals::nranks),
       nref(Globals::nranks), nderef(Globals::nranks), rdisp(Globals::nranks),
       ddisp(Globals::nranks), bnref(Globals::nranks), bnderef(Globals::nranks),
-      brdisp(Globals::nranks), bddisp(Globals::nranks) {
+      brdisp(Globals::nranks), bddisp(Globals::nranks),
+      pcoalesced_comms(std::make_shared<CoalescedComms>(this)),
+      do_coalesced_comms{
+          pin->GetOrAddBoolean("parthenon/mesh", "do_coalesced_comms", true)} {
```

Comment:
I currently set this to default to using the coalesced comms by default, mainly so that the tests would use the coalesced buffers. I am not sure we actually want this set to `true` by default, would like others opinions.

Reply:
I think default to true is reasonable, we already have some evidence that your solution produces improved performance at least outside of AthenaPK-style workflows

Reply:
Assuming it *runs* in riot and artemis I think default is true is probably correct. Makes sense to set it that way just to enable the tests.